app/src/main/res/values-zh/strings.xml

```diff
@@ -1,123 +1,196 @@
 
-	<string name="image_detail_dimensions">尺寸: %1$dx%2$d</string>
-	<string name="image_detail_size">文件大小: %1$s</string>
-	<string name="image_detail_modified">文件修改: %1$s</string>
-        <string name="toast_no_files">无文件显示</string>
+	<string name="toast_input_blocked">输入受阻。长按启用。</string>
+	<string name="toast_input_allowed">已启用输入</string>
 
-	<string name="long_loading_warning">%1$s 加载时间过长.</string>
-	<string name="long_loading_skipping">%1$s 加载时间过长. 跳过.</string>
+	<string name="long_loading_warning">加载 %1$s 花了很长时间</string>
+	<string name="long_loading_skipping">加载 %1$s 花了很长时间。正跳过。</string>
 	<string name="long_loading_skip_action">跳过</string>
 
+	<!-- Controls -->
+	<string name="controls_title">&appname; 操控</string>
+	<string name="controls_content1">观看幻灯片时，有两三个隐藏的操作手势。</string>
+	<string name="controls_content2">*左/右滑动*更改为下一张/上一张图片</string>
+	<string name="controls_content3">*单击*开始/停止幻灯放映</string>
+	<string name="controls_content5">*双击*暂停幻灯片放映而不显示详细信息面板</string>
+	<string name="controls_content4">*在幻灯片放映期间长按*禁用用户输入。这对防止你的奶奶/外婆在给设备除灰时不小心停止幻灯放映来说很有用。</string>
+
 	<!-- Image descriptions -->
-	<string name="image_description">照片</string>
-	<string name="logo_description">Logo</string>
+	<string name="image_description">图片</string>
+	<string name="logo_description">标识</string>
 
 	<!-- Credits -->
 	<string name="title_activity_credits">关于 &appname;</string>
-	<string name="credits_creator">由 <a href="https://michael.standen.link">&author;</a></string>
-	<string name="credits_content1">幻灯秀是一个在手机上看照片的简洁的应用。</string>
-	<string name="credits_content2">没有浮夸的装饰，本应用是市面上最简洁的幻灯片应用。</string>
-	<string name="credits_content3">在<a href="https://github.com/Screaminghawk/android-slideshow">Github</a>
-		上你可以查看源代码，报告故障，和提出建议。
+	<string name="credits_creator">by <a href="https://michael.standen.link">&author;</a></string>
+	<string name="credits_content1">&appname; 是一款查看你手机上图片的简单应用
+	程序</string>
+	<string name="credits_content2">没有花哨的功能, 此应用程序是市面上最简单的图片幻灯浏览
+	应用</string>
+	<string name="credits_content3">你可以在<a href="https://github.com/Screaminghawk/android-slideshow">Github</a>上
+	查看源代码、记录错误或请求功能。
 	</string>
-	<string name="credits_content4">这个应用软件完全免费且开源；遵循
-		<a href="https://tldrlegal.com/license/mit-license">MIT 许可证</a>。
+	<string name="credits_content4">此应用程序完全免费和开源，并在<a href="https://tldrlegal.com/license/mit-license">MIT 许可</a>下授权。
 	</string>
-	<string name="credits_content5">感谢<a href="https://github.com/psyanite">Yan Tsui</a>
-		对测试和设计的贡献&appname;icon.
+	<string name="credits_content5">感谢 <a href="https://github.com/psyanite">Yan Tsui</a>
+		测试并设计了&appname; 的图标。
 	</string>
-
-	<string name="credits_content6"><a href="https://github.com/wardbradt">Ward Bradt</a>
-		把&appname;从英文翻译成中文。&appname;的此版本翻译由<a href="https://github.com/Liuxi-G">Liuxi-G</a>
-		完成。
+	<string name="credits_content6"><a href="https://github.com/wardbradt">Ward Bradt</a>负责初版
+	简中翻译，<a href="https://github.com/Liuxi-G">Liuxi-G</a>进行了后续完善。
 	</string>
```

Comment:
This section is to give credits to the translators. Please add yourself to this list if you want to be credited in the app :)